db/src/queries/reports/reports_transaction_details.sql

```diff
@@ -1,76 +1,97 @@
+    csvgo.headline_artist_alt_genres,
+    mgo.headline_artist_main_genre
 FROM orders o
-         LEFT JOIN order_items oi ON (o.id = oi.order_id AND oi.item_type = 'Tickets')
-         LEFT JOIN order_items oi_fees ON (oi_fees.item_type = 'PerUnitFees' AND oi.id = oi_fees.parent_id)
-          LEFT JOIN order_items oi_event_fees
-                   ON (oi_event_fees.item_type = 'EventFees' AND o.id = oi_event_fees.order_id)
-         LEFT JOIN order_items oi_promo_code
-                   ON (oi_promo_code.item_type = 'Discount' AND oi.id = oi_promo_code.parent_id)
-         LEFT JOIN codes c ON oi.code_id = c.id
-         LEFT JOIN ticket_types tt ON (oi.ticket_type_id = tt.id)
-         LEFT JOIN (SELECT order_id,
-                           ARRAY_TO_STRING(ARRAY_AGG(DISTINCT p.payment_method), ', ') AS payment_method,
-                           ARRAY_TO_STRING(ARRAY_AGG(DISTINCT p.provider), ', ')       AS payment_provider
-                    FROM payments p
-                    WHERE p.status IN ('Completed','Refunded')
-                    GROUP BY p.order_id) AS p on o.id = p.order_id
-         LEFT JOIN (SELECT ti.order_item_id,
-                           NULLIF(ARRAY_TO_STRING(ARRAY_AGG(DISTINCT ti.check_in_source), ', '), '') AS check_in_source
-                    FROM ticket_instances ti
-                    GROUP BY ti.order_item_id) AS ti_agg ON ti_agg.order_item_id = oi.id
-         LEFT JOIN holds h on oi.hold_id = h.id
-         LEFT JOIN events e on oi.event_id = e.id
-         LEFT JOIN users u on coalesce(o.on_behalf_of_user_id, o.user_id) = u.id
+    LEFT JOIN order_items oi ON (o.id = oi.order_id AND oi.item_type = 'Tickets')
+    LEFT JOIN order_items oi_fees ON (oi_fees.item_type = 'PerUnitFees' AND oi.id = oi_fees.parent_id)
+    LEFT JOIN order_items oi_event_fees
+        ON (oi_event_fees.item_type = 'EventFees' AND o.id = oi_event_fees.order_id)
+    LEFT JOIN order_items oi_promo_code
+        ON (oi_promo_code.item_type = 'Discount' AND oi.id = oi_promo_code.parent_id)
+    LEFT JOIN codes c ON oi.code_id = c.id
+    LEFT JOIN ticket_types tt ON (oi.ticket_type_id = tt.id)
+    LEFT JOIN (SELECT order_id,
+        ARRAY_TO_STRING(ARRAY_AGG(DISTINCT p.payment_method), ', ') AS payment_method,
+        ARRAY_TO_STRING(ARRAY_AGG(DISTINCT p.provider), ', ')       AS payment_provider
+        FROM payments p
+        WHERE p.status IN ('Completed','Refunded')
+        GROUP BY p.order_id) AS p on o.id = p.order_id
+    LEFT JOIN (SELECT ti.order_item_id,
+        NULLIF(ARRAY_TO_STRING(ARRAY_AGG(DISTINCT ti.check_in_source), ', '), '') AS check_in_source
+        FROM ticket_instances ti
+        GROUP BY ti.order_item_id) AS ti_agg ON ti_agg.order_item_id = oi.id
+    LEFT JOIN holds h ON oi.hold_id = h.id
+    LEFT JOIN events e ON oi.event_id = e.id
+    LEFT JOIN users u ON COALESCE(o.on_behalf_of_user_id, o.user_id) = u.id
+    LEFT JOIN (SELECT csvg.headline_artist_alt_genres, csvg.artist_id, ea.event_id
```

Comment:
```suggestion
    LEFT JOIN (SELECT csvg.headline_artist_alt_genres, ea.event_id
```
You don't need to include columns that aren't used